lib/ffmpeg/encoderbase.py

```diff
@@ -184,6 +184,11 @@ def validate_caps(self):
         format_value(
           "{platform}.{driver}.slice > 1 unsupported in this mode", **vars(self)))
 
+    if vars(self).get("bframes", 2) > 0 and not self.caps.get("bframes", True):
```

Comment:
Why `2` as default?  If test does not set bframes, then should we assume 0?

Reply:
updated